Src/Sarif.PatternMatcher/AnalyzeCommand.cs

```diff
@@ -58,6 +80,37 @@ public AnalyzeCommand(IFileSystem fileSystem = null)
                     continue;
                 }
 
+                string name = definitions.ExtensionName;
+                string semanticVersion = null;
+                string version = null;
+
+                if (!string.IsNullOrEmpty(definitions.ValidatorsAssemblyName))
```

Comment:
>string [](http://example.com/codeflow?start=21&length=6)

hooray! extensions can now be versioned separately from the client tool. not sure we should do this, of course. :)